CRM/Remoteevent/RegistrationProfile.php

```diff
@@ -774,6 +774,7 @@ public function addDefaultContactValues(GetParticipantFormEventBase $resultsEven
                 ParticipantFormEventUtil::mapToPrefill($contact_data, $attribute_mapping, $resultsEvent, $value_callbacks);
             } catch (CiviCRM_API3_Exception $ex) {
                 // there is no (unique) primary email
+                // @todo CiviCRM_API3_Exception is an alias of CRM_Core_Exception so this catch is too broad.
```

Comment:
Maybe it isn't, we still want to catch all exceptions I guess.

https://github.com/civicrm/civicrm-core/blob/c26a7a076cb21df9bb79ff7cf57796be49184f4b/Civi/Api4/Generic/AbstractAction.php#L250-L251

Reply:
The comment `there is no (unique) primary email` expresses that the catch was intended for one special case, not to silently ignore any error.